package.json

```diff
@@ -40,7 +41,7 @@
       "ts"
     ],
     "testMatch": [
-      "__tests__/**/*.test.ts"
+      "**/*.test.ts"
```

Comment:
Is there any particular reason why this was changed? It seems like it'd be more efficient to narrow the directory down to `__tests__`

Reply:
I did not manage to make the tests run with this config, that's why I changed to match and find it. 
No clue why the first, with the prefixed `__tests__` directory didn't work =/

Reply:
Fair enough, marking as resolved